fail2ban/tests/servertestcase.py

```diff
@@ -74,14 +75,14 @@ def tearDown(self):
 		"""Call after every test case."""
 		self.server.quit()
 
-	def setGetTest(self, cmd, inValue, outValue=None, outCode=0, jail=None, repr_=False):
+	def setGetTest(self, cmd, inValue, outValue=(None,), outCode=0, jail=None, repr_=False):
 		setCmd = ["set", cmd, inValue]
```

Comment:
could you add a docstring describing parameters (please? ;) ) in particular why outValue is now a tuple